### ISSUE TITLE
suggestions to improve the Bayesian model

### ISSUE THREAD

**Issue by bnicenboim** [KKenez/Dyslexia_Paper#4]
1. Center and scale all the predictors that go into the horseshoe prior; right now, they have very different scales. (just make sure that you scale based on the training set and then you repeat the same scale for the other held-out dataset based on that (you can use `scale(...,  center = mean_training, scale = sd_training)`)
2. Use sum coding for the other predictors, that is one level 1 and the other -1, center Age, or center and scale it.
3.  Set priors for the intercept probably something like Normal(-2, 1) given that you expect ~10% dyslexia
4.  Set priors for the slopes of the other predictors. You can use the semi Bayesian ones recommended by Gelman (that we discussed in class).
5. You can use priorsense for a model without the "horseshoe predictors", to make sure that your priors for the effects of Age, Gender, etc are not overly informative.

But also see issue #1 , not sure what's going on with non natives, maybe we should do the analysis with only native speakers. There's still more data, right?

There's also another interesting model which is like the previous, but the horseshoe parameters include the game-related coefficients and the interaction with the demographic data, that is `b ~ ` is the same and `a ~` is:
` ~ Clicks1 + Hits1 + Misses1 + Score1 + Accuracy1 + Missrate1 +  Clicks1:Age + Hits1:Age + Misses1:Age + Score1:Age + Accuracy1:Age + Missrate1:Age + Clicks1:Gender + Hits1:Gender  + Misses1:Gender  + Score1:Gender  + ....` (Maybe create the formula programmatically).
In this model, Age should also be centered and scaled.

Additional suggestions are to set the `par_ratio` argument of the horseshoe prior to different values and see if it makes a difference.

You can compare all the Bayesian models in the validation set with `loo`
